datavault-common/pom.xml

```diff
@@ -156,6 +156,12 @@
             <artifactId>vault-java-driver</artifactId>
             <version>3.1.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>16.0.3</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Not sure how it got there, looks like a intellij thing, we might not need it.

Reply:
https://mvnrepository.com/artifact/org.jetbrains/annotations/13.0 I don't think we need it unless you've added some fancy comment!